jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/domain/mappers/ForecastDataMapper.kt

```diff
@@ -27,14 +28,10 @@ class ForecastDataMapper {
             iconUrl = generateIconUrl(forecast.weather[0].icon)
         }
 
-        return ModelForecast(convertDate(forecast.dt), description,
+        return ModelForecast(forecast.dt, description,
                 forecast.temp.max.toInt(), forecast.temp.min.toInt(), iconUrl)
     }
 
     private fun generateIconUrl(iconCode: String) = "http://openweathermap.org/img/w/$iconCode.png"
```

Comment:
Leave it as is. But an alternative could be to put this in a getter of the `iconUrl` property